### DISCUSSION TITLE
Opening :Git split in at the bottom of the window instead of the top

### DISCUSSION THREAD

**Discussion by kelly-lin** [tpope/vim-fugitive#1873] [Q&A] [answered]
Is there a way to spawn the window split made by :Git at the bottom of the window instead of the top?

**Comment by tpope** [comment, marked as answer]
`:below Git`.

**Reply by kelly-lin** [reply]
Thanks!

**Comment by francis-junr** [comment]
How do you make that persist?

**Reply by tpope** [reply]
`:Git` respects the `'splitbelow'` option.

**Reply by francis-junr** [reply]
Thank you!
